packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -208,6 +211,10 @@ const onRenderParticipantDefault = (
       /* @conditional-compile-remove(PSTN-calls) */
       participantState={callingParticipant.state}
       ariaLabelledBy={participantAriaLabelledBy}
+      /* @conditional-compile-remove(raise-hand) */
+      showCoinBorder={callingParticipant?.raisedHand ? true : false}
+      /* @conditional-compile-remove(raise-hand) */
+      coinBorderColor={callingPalette.raiseHandGold}
```

Comment:
I don't think we need to add this coinBorderColor prop for ParticipantItem. We can just change the styles prop (on line 199) to have a border with this color when `callingParticipant?.raisedHand` is true

Reply:
thanks, i've added styles using avatar function parametsers